e2e/nomostest/wait_for_sync.go

```diff
@@ -133,13 +125,6 @@ func (nt *NT) WatchForAllSyncs(options ...WatchForAllSyncsOptions) error {
 		}
 	}
 
-	watchOptions := []testwatcher.WatchOption{
-		testwatcher.WatchTimeout(opts.timeout),
```

Comment:
Did this default timeout get lost?

Reply:
No. The default timeout is also configured directly on the Watcher struct.